src/sentry/integrations/slack/message_builder/notifications.py

```diff
@@ -0,0 +1,32 @@
+import re
+from urllib.parse import urljoin
+
+from sentry.integrations.slack.utils import LEVEL_TO_COLOR
+
+
+def build_notification_footer(links, notification_type):
+    referrer = re.sub("Notification$", "Slack", notification_type)
+    settings_url = urljoin(links["settings_url"], "?referrer=" + referrer)
+
+    if notification_type == "ReleaseActivityNotification":
```

Comment:
Comparing against class name can be flakey because the names can change. You should instead compare against the value returned by `.get_category()`.